galaxy_ie_helpers/__init__.py

```diff
@@ -28,7 +28,7 @@ def _get_ip():
     p2 = subprocess.Popen(cmd_grep, stdin=p1.stdout, stdout=subprocess.PIPE)
     cmd_awk = ['awk', '{ print $2 }']
     p3 = subprocess.Popen(cmd_awk, stdin=p2.stdout, stdout=subprocess.PIPE)
```

Comment:
```suggestion
    p3 = subprocess.Popen(cmd_awk, stdin=p2.stdout, stdout=subprocess.PIPE, text=True)
```

@sneumann can you please test if this also works? That looks a bit cleaner to me.